presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -73,6 +76,8 @@
     private final DataSize outputDataSize;
     private final long outputPositions;
 
+    private final List<OperatorStats> operatorSummaries;
```

Comment:
What I meant is, do we roll them up? Or do we still return one instance per operator per task?